daniel-alejandro-garcia/tarea-5/js/slidePuzzle.js

```diff
@@ -0,0 +1,105 @@
+let buttonContainer = document.getElementById('tiles');
+let matrizSize = 4;
+let numberOfTiles = matrizSize ** 2;
+let bottonSelected = numberOfTiles;
+let shuffled = false;
+
+newGame();
+
+function newGame() {
+    loadTiles(matrizSize);
+}
+
+function loadTiles() {
+    for (let id = 1; id <= numberOfTiles; id++) {
+        let newTile = document.createElement('button');
+        newTile.id = `btn${id}`;
+        newTile.setAttribute('index', id);
+        newTile.innerHTML = id;
+        newTile.classList.add('btn');
+        newTile.addEventListener('click', function () {
+            swap(parseInt(this.getAttribute('index')));
+        });
+        buttonContainer.append(newTile);
+    }
+    selectedTileId = 'btn' + bottonSelected;
+    selectedTile = document.getElementById(selectedTileId);
+    selectedTile.classList.add("selected");
+}
+
+function shuffle() {
+    let minShuffles = 100;
+    let totalShuffles = minShuffles + Math.floor(Math.random() * 200);
+    for (let i = minShuffles; i <= totalShuffles; i++) {
```

Comment:
@da-garcia , please use modern approach like forEach, map or reduce and remember proper naming for variable i